src/Middleware/IdleConnectionMiddleware.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\Middleware;
+
+use ArrayObject;
+use Doctrine\DBAL\Driver;
+use Doctrine\DBAL\Driver\Middleware;
+use Symfony\Bridge\Doctrine\Middleware\IdleConnection\Driver as IdleConnectionDriver;
+
+class IdleConnectionMiddleware implements Middleware, ConnectionNameAwareInterface
```

Comment:
I'm confused why is this class here. Wasn't the middleware moved to bridge?

Reply:
At the moment, that's not possible without first moving ConnectionNameAwareInterface to the bridge.